### ISSUE TITLE
can't import torchvision exported shufflenet

### ISSUE THREAD

**Issue by thehaigo** [elixir-nx/axon_onnx#32]
torchvision exported densenet and inceptionv3
import error  `unable to build model from ONNX graph`

shufflenet export python code
```python
import torchvision
import torch

net = torchvision.models.shufflenet_v2_x1_0(pretrained=True)
model_onnx_path = "model/shufflenet/model.onnx" 
input_names = [ "input" ] 
output_names = [ "output" ] 

input_shape = (3, 224, 224)
batch_size = 1
dummy_input = torch.randn(batch_size, *input_shape) 

output = torch.onnx.export(net, dummy_input, model_onnx_path, \
                   verbose=False, input_names=input_names, output_names=output_names)

```

Elixir import code

```
{model, params} = AxonOnnx.import("model/shufflenet/model.onnx")
** (ArgumentError) unable to build model from ONNX graph, expected value onnx::Conv_378 to be a graph input, but it was not present in built graphs
    (axon_onnx 0.1.0) lib/axon_onnx/deserialize.ex:1673: AxonOnnx.Deserialize.axon!/2
    (axon_onnx 0.1.0) lib/axon_onnx/deserialize.ex:743: AxonOnnx.Deserialize.recur_nodes/2 
    (elixir 1.13.0) lib/enum.ex:2396: Enum."-reduce/3-lists^foldl/2-0-"/3
    (axon_onnx 0.1.0) lib/axon_onnx/deserialize.ex:42: AxonOnnx.Deserialize.graph_to_axon/2
    (axon_onnx 0.1.0) lib/axon_onnx/deserialize.ex:25: AxonOnnx.Deserialize.to_axon/2
```

**Comment by seanmor5** [comment]
This has recently been fixed along with several other torchvision models. Please give it a try and let me know if you have problems :)
